faust/app/base.py

```diff
@@ -984,13 +992,15 @@ def _on_rebalance_when_stopped(self) -> None:
                 await self.consumer.pause_partitions(assigned)
```

Comment:
I forget why we were pausing the consumer for all partitions. Was it only for the tables? If yes, this might not be needed anymore right? Further, at this point the fetcher should be stopped so we shouldn't be reading messages anyway?

Reply:
We are no longer stopping the fetcher, so here we are pausing all partitions so that recovery can start.

Recovery will first resume active table partitions, then when recovery is finished it will resume all partitions except for active table partitions.